jwst/wfs_combine/wfs_combine_step.py

```diff
@@ -69,7 +72,8 @@ def process(self, input_table):
             output_model.meta.cal_step.wfs_combine = 'COMPLETE'
             output_model.meta.asn.pool_name = asn_table['asn_pool']
             output_model.meta.asn.table_name = os.path.basename(input_table)
-            output_model.meta.filename = which_set['name']
+            # format the filename here
+            output_model.meta.filename = which_set['name'].format(suffix=self.suffix) + self.output_ext
```

Comment:
I assume `step.output_ext` defaults to "fits"?

Reply:
Yup!

https://github.com/spacetelescope/jwst/blob/e1b66cc4c8c8d2cafd2be4828de7de67184f56a6/jwst/stpipe/core.py#L23

I ran just the `jwst.regtest.test_nircam_wfs` with this updated PR and these tests all passed:
https://plwishmaster.stsci.edu:8081/blue/organizations/jenkins/RT%2FJWST-Developers-Pull-Requests/detail/JWST-Developers-Pull-Requests/1503/tests

Let me know if another full regtest run is called for.

Reply:
Full regtest running: https://plwishmaster.stsci.edu:8081/job/RT/job/JWST-Developers-Pull-Requests/1504/